src/Umbraco.Web/Models/Mapping/MemberModelMapper.cs

```diff
@@ -96,9 +98,10 @@ public override void ConfigureMappings(IConfiguration config, ApplicationContext
                 .ForMember(dto => dto.HasPublishedVersion, expression => expression.Ignore());
 
             //FROM MembershipUser TO MemberBasic
-            config.CreateMap<MembershipUser, MemberBasic>()
+            config.CreateMap<MembershipUser, MemberBasic>()                
                 //we're giving this entity an ID of 0 - we cannot really map it but it needs an id so the system knows it's not a new entity
                 .ForMember(member => member.Id, expression => expression.MapFrom(user => int.MaxValue))
+                .ForMember(display => display.Udi, expression => expression.Ignore())

```

Comment:
When we map from IMember we will map UDI, however from MembershipUser there is no known ID. A MembershipUser is a model representing the object coming from a MembershipProvider and since this can be replaced by anything we don't know what kind of ID it's going to have. In any case, we won't be dealing with UDIs for MembershipUsers